src/Components/Web/src/Virtualization/Virtualize.cs

```diff
@@ -88,6 +88,15 @@ public sealed class Virtualize<TItem> : ComponentBase, IVirtualizeJsCallbacks, I
         [Parameter]
         public ICollection<TItem>? Items { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value that determines how many additional items will be rendered
+        /// before and after the visible region. This help to reduce the frequency of rendering
+        /// during scrolling. However, higher values mean that more elements will be present
+        /// in the page.
+        /// </summary>
+        [Parameter]
+        public int OverscanCount { get; set; } = 3;
```

Comment:
@MackinnonBuck What do you think about including this to give developers an extra dial to turn? During my experiments I found it was rendering at a higher frequency than I liked during scrolling, doing quite a bit of CPU work (and in the Blazor Server case, causing a lot of network traffic to get small slices of data).

Reply:
@SteveSandersonMS This sounds like a great idea to me.

Reply:
OK cool. It's a very small and low-risk addition so let's include it.